src/components/app/Details.css

```diff
@@ -4,6 +4,17 @@
   flex-direction: column;
 }
 
+/* contains the tabbar and, if present, the sidebar button */
+.Details-top-bar {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between; /* This pushes the sidebar button to the right */
+  border: solid var(--grey-30);
+  border-width: 1px 0 1px;
+  background: var(--grey-10);
+}
+
```

Comment:
This works for me, if you move this back to `TabBar.css` and use a more generic name, such as `.tabBarContainer-top`, so that it's easily reusable.

In the future patch where you'll implement a bottom tab bar, you may split it in 2: `.tabBarcontainer` with common values, and `.tabBarContainer-top/bottom` for the specific ones (I believe that would especially be the borders?).

Reply:
I disagree. The TabBar now only includes the tabs and not the entire bar. And the `Details-top-bar` element is created by `Details.js`. I think it would make more sense to keep this in Details.css.

Reply:
The rationale is that the TabBar component exposes reusable CSS style rules that work with it.

Reply:
Also when you'll implement the TabBarTop component this will move back there anyway. Or I'm missing something :-)

Reply:
It seems we've had different things in mind.

For what I'm imagining, maybe `Tabs` would be a better component name than `TabBar`.

How about this:

```
<DetailsTopBar>
  <Tabs mode="top"/>
  <OptionalSidebarButton/>
</Details>
```

The whole point of this changeset is that, since there is a sidebar button which has nothing to do with tabs, the "bar" that contains both tabs and the sidebar button should not be considered a part of the "tabs" infrastructure. The "bar" is outside the tabs. And I don't see a need for a reusable bar, but I can make a `DetailsTopBar` if it helps.

Reply:
This would work for me!